src/test/java/net/openhft/chronicle/queue/internal/reader/ChronicleMethodReaderTest.java

```diff
@@ -187,10 +187,12 @@ public void shouldFilterByInclusionRegex() {
                 .forEach(msg -> assertThat(msg, containsString("goodbye")));
     }
 
-    @Ignore("https://github.com/OpenHFT/Chronicle-Queue/issues/1150")
     @Test
     public void shouldFilterByMultipleInclusionRegex() {
-        basicReader().withInclusionRegex(".*bye$").withInclusionRegex(".*o.*").execute();
+        basicReader()
+                .withInclusionRegex(".*bye.*")
```

Comment:
@yevgenp does this include OR or AND? We should specify in the javadoc what happens if you specify multiple inclusion regexes.

Reply:
AND

Reply:
@tgd I have added javadoc. Please squash/merge if you are happy with this